tests/functional/commands/validate/lib/test_sam_template_validator.py

```diff
@@ -7,216 +8,13 @@
 from samcli.commands.validate.lib.sam_template_validator import SamTemplateValidator
 from samcli.commands.validate.lib.exceptions import InvalidSamDocumentException
 
+# Out of scope because https://stackoverflow.com/a/47224266
```

Comment:
```suggestion
# Out of TestValidate's scope because https://stackoverflow.com/a/47224266
```

Reply:
If I did not have the context, I would not get what "out of scope" means here.. out of this PR's scope or out of this file's scope 😂